src/index.js

```diff
@@ -19,10 +19,28 @@ class Dropzone extends React.Component {
     };
   }
 
+  componentWillMount() {
+    if (this.props.global && typeof document) {
+      document.addEventListener('dragenter', this.onDragEnter);
```

Comment:
You assign event listeners to some class methods that doesn't exist. This will not work I think.